be2-scala/build.sbt

```diff
@@ -9,7 +9,7 @@ version := "0.1"
 
 scalaVersion := "2.13.5"
 
-
+parallelExecution in ThisBuild := false
```

Comment:
I don't if we should keep this. but it's very useful to keep it as it helps to debug the tests/ right @RaulinN ?